### ISSUE TITLE
[SEE COMMENTS FOR FIX] 500 error after update to 1.7.6 if updated right away.

### ISSUE THREAD

**Issue by bodie007** [CalderaWP/Caldera-Forms#2896]
## Do You Need Immediate Help?
* If so, please open a support ticket at https://CalderaForms.com/support
no
## Is This A Feature Request?
* If so, that's fine, please describe your desired featured and delete the rest of this template.
no

## What Version Of Caldera Forms, WordPress and PHP Are You Using?
*You can copy the "Short Debug" line from the Caldera Forms support menu's debug tab
Version: 1.7.6
## Does Your Issue Persist When You Disable All Other Plugins and Switch To The Default Theme?
not tried
## What Is The Unexpected Behaviour?
* Please explain what you did, what you expected to happen, and what happened instead.
[10-Jan-2019 18:22:56 UTC] PHP Fatal error:  Interface 'Monolog\ResettableInterface' not found in /home/johnmini17/public_html/wp-content/plugins/caldera-forms/vendor/monolog/monolog/src/Monolog/Logger.php on line 29
## What PHP Errors Have You Logged While Reproducing This Bug?
* See: https://calderaforms.com/2016/05/wordpress-debug-logging/
none
## What JavaScript Errors Have You Seen While Reproducing This Bug?
* See: http://webmasters.stackexchange.com/questions/8525/how-to-open-the-javascript-console-in-different-browsers
none

**Comment by Shelob9** [comment]
TL;DR For users with this bug - Sorry, please delete Caldera Forms and reinstall form WordPress.org

TL;DR For @New0 We need to discuss #2875 again. I'm meeting with Pete Shum to get us onboarded to the Branch beta, which automates this process on January 30th.

Explanation:

I just installed using WordPress plugin installer a new poopy.life site, no issues: `WordPress Version: 4.9.9PHP Version: 5.6.37MySQL Version: 5.5.56Caldera Forms Version: 1.7.6WP_DEBUG:`

I noticed that is WordPress 4.9.9,  update to 5.0.3, no issues with Caldera Forms and reinstalled Caldera Forms. I deleted Caldera Forms and then reinstalled Caldera Forms through WordPress plugin installer - no issues.

So I think that because I pushed to trunk, then my final QA caught that #2875 issue which only happens at build time -- tests had passed with a ZIP file of what I was planning to ship to WordPress.org, so I fixed it in trunk and then did the tag. No one should have gotten that version from trunk since I had not changed the tag yet, I used WP Rollback to get it on my test site. I think that @bodie007 and @jeroenrotty (#2897) got the update from trunk, which WordPress.org should not do, but their caching is odd.

**Comment by jeroenrotty** [comment]
Hi Josh,

By reinstall, you just mean deactivate it and reinstall from .org? Just tried that, didn't make a difference...

J

**Comment by Shelob9** [comment]
BTW It's also right on trunk, so this gets into my theory that hosts or someone slightly higher up the chain are caching plugin update files. Otto has told me before that WordPress.org isn't caching those ZIP builds, but I don't know. Also, I just downloaded from WordPress.org and checked the file and its fine.

FWIW This is the other reason besides security why WordPress should be checking file hashes before installing a package.

**Comment by Shelob9** [comment]
@jeroenrotty Here is what I got from https://downloads.wordpress.org/plugin/caldera-forms.1.7.6.zip -
https://drive.google.com/file/d/1VprcPZVRjnNxRYviacMIRqhYAJCj2Y_d/view?usp=sharing
